tracer/test/test-applications/security/Samples.Security.AspNetCore2/Samples.Security.AspNetCore2.csproj

```diff
@@ -37,5 +37,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
+    <ProjectReference Include="..\..\..\..\src\Datadog.Trace.Annotations\Datadog.Trace.Annotations.csproj" />
```

Comment:
Urgh, I didn't notice that Samples.Security is directly referencing Datadog.Trace 😭 We're going to have to tear that out ASAP. We shouldn't be doing this (for various reasons), and instead should be relying on indirect references or Otel, like we do in our other (non-security) integration tests

Reply:
Noted for future PR